feat(HTTP): Refresh 5XX response pages (#35361)
mdn/content: files/en-us/web/http/status/510/index.md

{{HTTPSidebar}}

> The HTTP **`510 Not Extended`** [server error response](/en-US/docs/Web/HTTP/Status#server_error_responses) status code is sent in the context of the HTTP Extension Framework ({{RFC("2774")}}).
> A client may send a request that contains an extension declaration that describes the extension to be used to process a request.
> If the server receives such a request, but any described extensions are not supported, the server responds with a `510` status.

Reviewer: Again, is it better to say what it does first, or what the context is? I prefer the error

```suggestion
The HTTP **`510 Not Extended`** [server error response](/en-US/docs/Web/HTTP/Status#server_error_responses) status code is sent when the client request declares an HTTP Extension ({{RFC("2774")}}) that should be used to process the request, but the extension is not supported.
```

Author: More to the point, I guess 👍🏻
